apps/wallet-mobile/src/features/Transactions/TxHistoryNavigator.tsx

```diff
@@ -161,6 +161,8 @@ export const TxHistoryNavigator = () => {
                       title: meta.name,
                       headerTransparent: true,
                       headerRight: headerRightHistory,
+                      // marginRight override to compensate for the headerRight
```

Comment:
Because this headerRight is wider than the back button, adds extra margin to the tag container to compensate the asimetry